fmriprep/cli/parser.py

```diff
@@ -336,13 +337,12 @@ def parse_args(args=None, namespace=None):
     # Retrieve logging level
     build_log = config.loggers.cli
 
-    if config.execution.fs_license_file is None:
+    if not check_valid_fs_license(lic=config.execution.fs_license_file):
         raise RuntimeError("""\
 ERROR: a valid license file is required for FreeSurfer to run. fMRIPrep looked for an existing \
 license file at several paths, in this order: 1) command line argument ``--fs-license-file``; \
 2) ``$FS_LICENSE`` environment variable; and 3) the ``$FREESURFER_HOME/license.txt`` path. Get it \
 (for free) by registering at https://surfer.nmr.mgh.harvard.edu/registration.html""")
-    os.environ['FS_LICENSE'] = str(config.execution.fs_license_file)
```

Comment:
Are we positive this is not necessary anymore? It is outside the if branch.

Reply:
we'll monitor prior to the 20.1 release, but this should be good